domain/src/ValueObjects/Transaction.php

```diff
@@ -178,17 +233,17 @@ public function __toString(): string
             $this->date->__toString(),
             $this->operation->value,
             $this->isIncome ? 'yes' : 'no',
-            $this->costBasis->__toString(),
+            $this->marketValue?->__toString() ?? 'N/A',
```

Comment:
ℹ️ By the way, "cost basis" has been renamed for "market value" because depending on the operation (e.g. send), the cost basis is actually the proceeds. In all cases however, it is indeed the market value of the cost basis and/or the proceeds that we are reporting